app/src/main/java/com/onboarding/parkingsystemjava/entity/Reservation.java

```diff
@@ -10,7 +10,7 @@
     private String userPassword;
 
     public Reservation() {
-        this.parkingLot = - ConstantUtils.INT_ONE;
+        this.parkingLot = ConstantUtils.INT_MINUS_ONE;
```

Comment:
int minus is not a representative name, what is the purpose of the const?

Reply:
In this case is assigned -1 and the verifier checks this value to tell if the user has the parking lot EditText, and the months in Calendar starts in 0  so the const is used to subtract 1 from the month value of the date picker.

Reply:
It would be nice that the name represent this

Reply:
done